### ISSUE TITLE
audio potentially allows mimicking Voice interaction

### ISSUE THREAD

**Issue by chaals** [w3c/html#1312]
The audio element potentially allows mimicking of a voice user intereface such as a screen reader or voice assistant. This should be noted in security and privacy sections as appropriate. see also https://github.com/WebAudio/web-audio-api/issues/1487

**Comment by chaals** [comment]
Add a short note, include mitigation strategy like disabling `autoplay`...

**Comment by chaals** [comment]
closed in #1580
